package.json

```diff
@@ -89,7 +89,10 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
+    "@fast-csv/parse": "^4.3.0",
```

Comment:
Are we using all 3 pkgs?

Reply:
I can get rid of the packages that im not using

Reply:
I'll get rid of any unused packages that I added lol